### ISSUE TITLE
Functionality tests

### ISSUE THREAD

**Issue by davidfetter** [trlorenz/PG-recursively_delete#7]
It'd be nice to have some tests included in the repo, not least because any refactor would have those as a start when figuring out whether the code still works. I'm thinking a not-huge schema and a not-huge number of rows in same, but exercising all the features the code covers.

**Comment by trlorenz** [comment]
Yes. (Again: Ad hoc one-offs.) Do you have a preferred example of PG test boilerplate I could look at?

**Comment by davidfetter** [comment]
I put a teensy bit in for smoke tests of today's PR :)

**Comment by davidfetter** [comment]
If that schema from the docs is OK to use, obviously with phony data, that'd be handy.

**Comment by trlorenz** [comment]
You talking about the one I used for the preview in the readme? That's not available to me anymore, but I think I can fake it up. Give me a couple days.

**Comment by davidfetter** [comment]
Thanks!

**Comment by trlorenz** [comment]
https://gist.github.com/trlorenz/98a277e336caea4501e60c21d316d0b1

Should be an ok place to start. Deleting record #1 from recdel_users hits most of the weird cases.
